frontend/packages/helm-plugin/src/utils/helm-utils.ts

```diff
@@ -175,7 +175,7 @@ export const getOriginRedirectURL = (
 ) => {
   switch (actionOrigin) {
     case HelmActionOrigins.topology:
-      return `/topology/ns/${namespace}?view=graph`;
+      return `/topology/ns/${namespace}`;
     case HelmActionOrigins.list:
```

Comment:
do we need the case for `HelmActionOrigins.list` now?

cc @rohitkrai03

Reply:
Yeah, that's for helm list page.